tests/test_compatibility_file.py

```
@@ -0,0 +1,75 @@
+
+    def test_calc_md(self):
+        structure = bulk("Al", cubic=True).repeat([2, 2, 2])
+        potential = "1999--Mishin-Y--Al--LAMMPS--ipr1"
+        units = "metal"
+        calc_kwargs = {
+            "temperature": 500.0,
+            "pressure": 0.0,
+            "n_ionic_steps": 1000,
+            "n_print": 100,
+        }
+        shell_output, parsed_output, job_crashed = lammps_file_interface_function(
```

Ruff (0.14.5)

Unpacked variable `shell_output` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)